thoth/metrics_exporter/jobs/db.py

```diff
@@ -38,10 +35,22 @@ class DBMetrics(MetricsBase):
     """Class to evaluate Metrics for Thoth Database."""
 
     _METRICS_EXPORTER_INSTANCE = os.environ["METRICS_EXPORTER_INFRA_PROMETHEUS_INSTANCE"]
+    _MANAGEMENT_API_INSTANCE = os.environ["MANAGEMENT_API_PROMETHEUS_INSTANCE"]
+    _INVESTIGATOR_INSTANCE = os.environ["INVESTIGATOR_PROMETHEUS_INSTANCE"]
+    _USER_API_INSTANCE = os.environ["USER_API_PROMETHEUS_INSTANCE"]
 
     _SCRAPE_COUNT = 0
     _BLOAT_DATA_SCRAPE_INTERVAL_DAYS = 7
 
+    _TABLE_COMPONENT_USING_DATABASE = {
+        "user-api": {"uses_pushgateway": False, "instance": _USER_API_INSTANCE},
+        "management-api": {"uses_pushgateway": False, "instance": _MANAGEMENT_API_INSTANCE},
+        "investigator": {"uses_pushgateway": False, "instance": _INVESTIGATOR_INSTANCE},
+        "package-releases": {"uses_pushgateway": True, "env": Configuration.DEPLOYMENT_NAME},
+        "graph-refresh": {"uses_pushgateway": True, "env": Configuration.DEPLOYMENT_NAME},
+        "graph-sync": {"uses_pushgateway": True, "env": Configuration.DEPLOYMENT_NAME},
```

Comment:
Is this something that needs to maintenance? I'm wondering if this could be queried (e.g. based on labels).

Reply:
Yes it is actually possible, we can use "env" label to differentiate between environments and the env variable about distance are not required anymore. It requires adding that label to components, because right now only in component pushing to pushegateway was present because instance is not set in that case. Let's do that then! Thanks @fridex !